project.clj

```diff
@@ -3,12 +3,37 @@
   :url "https://github.com/fredericksgary/test.chuck"
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [[org.clojure/clojure "1.6.0"]
+  :dependencies [[org.clojure/clojure "1.7.0"]
+                 [org.clojure/clojurescript "0.0-3308"]
```

Comment:
Bumped to clojurescript 1.7.48

Also added `:scope "provided"` to both clj & cljs. I checked and it works fine in that it doesn't install clojurescript as a transitive dependency when declaring the test.chuck dependency. And being strict, cljs is not a dev dependency, it's a "core" dependency, it's just that it's not always used, right?

But anyways, I'm okay if you still prefer to move it to `:dev`, just let me know.